securedrop_client/api_jobs/downloads.py

```diff
@@ -53,7 +53,7 @@ def call_api(self, api_client: API, session: Session) -> Any:
         # TODO: Once https://github.com/freedomofpress/securedrop-client/issues/648, we will want to
         # pass the default request timeout to api calls instead of setting it on the api object
         # directly.
-        api_client.default_request_timeout = 20
+        api_client.default_request_timeout = 120
```

Comment:
so you'll notice we have an issue for acquiring realistic timeouts for api calls here: https://github.com/freedomofpress/securedrop-client/issues/648

if you read through john's findings you'll see that 20 seconds is cutting it close for `get_sources` when there's 200, but it looks to be more than enough time for the other API calls made during a sync: `get_submissions` and `get_all_replies`

eventually we want to be able to scale these timeouts the way we do with file downloads but for now, using a default of 20 seconds seems like plenty of time for most cases except when there are 200+ sources for `get_sources` 

Unfortunately we don't yet support passing a different default timeout to just the `get_sources` api method as you can see from my comment above:

```
        # TODO: Once https://github.com/freedomofpress/securedrop-client/issues/648, we will want to
        # pass the default request timeout to api calls instead of setting it on the api object
        # directly.

```
i think until we can vary our timeouts per request and based on size of the request, we should use the largest measurement from john's profiling and mutliple it by 1.5 or 2, so I think 30-40 second timeouts should be enough, right?

Reply:
I am wondering if @ntoll tested with https://github.com/freedomofpress/securedrop/pull/5100 which should already significantly improve performance (the cache gets primed on app server start)